src/Renci.SshNet/NetConfClient.cs

```diff
@@ -21,8 +21,8 @@ public class NetConfClient : BaseClient
         /// Gets or sets the operation timeout.
         /// </summary>
         /// <value>
-        /// The timeout to wait until an operation completes. The default value is negative
-        /// one (-1) milliseconds, which indicates an infinite time-out period.
+        /// The timeout to wait until an operation completes. The default value is <see cref="SshNet.Session.InfiniteTimeSpan"/>

```

Comment:
I do not want to expose them.
The BCL already defines a **Timeout** class with these constants.
Do not that this class is (may not be, to verify) not available on all our supported target frameworks.